java/src/main/native/src/ColumnViewJni.cpp

```diff
@@ -2452,7 +2452,14 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnView_getJSONObject(
     options.set_allow_single_quotes(allow_single_quotes);
     options.set_strip_quotes_from_single_strings(strip_quotes_from_single_strings);
     options.set_missing_fields_as_nulls(missing_fields_as_nulls);
-    return release_as_jlong(cudf::get_json_object(n_strings_col_view, *n_scalar_path, options));
+    std::unique_ptr<cudf::column> result_col_ptr;
+    try {
```

Comment:
@ttnghia Implemented your suggestion and addressing the exception handling on the JNI side.